packages/connect-web/package.json

```diff
@@ -45,7 +45,8 @@
     "dependencies": {
         "@trezor/connect": "workspace:*",
         "@trezor/connect-common": "workspace:*",
-        "@trezor/utils": "workspace:*"
+        "@trezor/utils": "workspace:*",
+        "crypto-browserify": "^3.12.0"
```

Comment:
dependency or devDependency ?  🤔

Reply:
connect-web itself only imports scheduleAction, createDeferred and createDeferredManager